### DISCUSSION TITLE
Recommended Cable Lengths

### DISCUSSION THREAD

**Discussion by Nexis81** [netbox-community/netbox#17887] [Ideas]
Is there any way to develop some sort of recommended cable length module to help with rack planning. Where it would look at the rack position of cable end devices and try to figure out how long of a cable you may need like 1, 2, 3, 5, or 7 ft.

**Comment by chris240189** [comment]
That's not an easy task as you would need to model the physical interface position for every device and also need to model cable pathing within the rack through cable management (not even talking about throwing cables from rack to rack over the racks or under the floor).

**Comment by candlerb** [comment]
Maybe this is the answer, when it's released: https://store.steampowered.com/app/2640920/Network_Engineer_Simulator/ :-)
